charts/clickhouse/Chart.yaml

```diff
@@ -2,16 +2,17 @@ apiVersion: v2
 name: clickhouse
 description: A Helm chart for ClickHouse
 type: application
-version: 23.10.1
+version: 23.11.0
 appVersion: "22.8.8"
 icon: https://raw.githubusercontent.com/ClickHouse/ClickHouse/master/website/images/logo-400x240.png
 sources:
   - https://github.com/signoz/charts/
   - https://github.com/ClickHouse/ClickHouse
 dependencies:
   - name: zookeeper
-    repository: https://raw.githubusercontent.com/bitnami/charts/pre-2022/bitnami
-    version: 7.0.5
+    repository: oci://registry-1.docker.io/bitnamicharts
```

Comment:
How does this `oci::` work?

Reply:
OCI (Open Container Initiative) is a project that defines open standards for container formats and runtime.
Big companies and cloud platforms are their members.

Back in January, Bitnami introduced support for OCI and they plan to eventually moving out of _index.yaml_ chart release.
https://blog.bitnami.com/2023/01/bitnami-helm-charts-available-as-oci.html

For the end-user, there would not be any significant change.
You can find the OCI-based charts repository in DockerHub: https://hub.docker.com/r/bitnamicharts/zookeeper

Reply:
Do all helm versions support oci scheme?

Reply:
@srikanthccv It supports starting from Helm v3.8.0.